### DISCUSSION TITLE
having trouble finding solutions using Stipple documentation

### DISCUSSION THREAD

**Discussion by ghost** [GenieFramework/Stipple.jl#140] [Q&A] [answered]
Hi all
I am REALLY enjoying using Stipple and it's teaching me lots about julia in general. But I am having a little trouble finding solutions or documentation for things I KNOW that Stipple will do. I am coming from python so have some existing things I want to reproduce ( see below) and I want to know the best way to find documentation OR an example of things like "filter_action" in the python dash code below. All it does is allow me to set a filter at the head of each column on a dash board. It's pretty useful as it's free form. How would you go about finding the Stipple approach to this? I don't want the fish but I would appreciate the fishing rod more.
thanks
theakson

```

dash_table.DataTable(
                                   id='datatable_id',
                                   columns=[{"name": i, "id": i} for i in df_today.columns],
data=df_today.to_dict('records'),
                                   editable=False,
                                   filter_action="native",
                                   sort_action="native",
                                   sort_mode="multi",
                                   row_selectable="multi",
                                   row_deletable=False,
                                   selected_rows=[],
                     ##            page_action="native",
                     ##            page_current= 0,
                     ##            page_size= 10,
                                   style_cell={'textAlign': 'left','width': '80px'},
                                   fixed_rows={ 'headers': True, 'data': 0 },
                                   style_table={ 'maxHeight': '300px','overflowY': 'scroll' },         # https://dash.plotly.com/datatable/sizing
                                                       )
                                 ]
                                 ,className='row'), 

```

**Comment by hhaensel** [comment, marked as answer]
This should be the fishing rod
https://v1.quasar.dev/vue-components/table

**Reply by ghost** [reply]
@hhaensel  I SWEAR this is true, I just watched this https://youtu.be/xlieoONTLUs?t=6867  :-)  I was ALMOST there!!! thank you for the rod, I'm off to the quasar pond next week to see what I can catch. Thanks again for the help and such a stunningly great package. 
theakson
